web/src/lib/components/alert/alert.tsx

```diff
@@ -0,0 +1,49 @@
+import { h, Component } from 'preact';
+import Icon from '../icon';
+
+interface Props {
+  text: string;
+  active: boolean;
+}
+
+interface State {
+  active: boolean;
+}
+
+export default class Alert extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+
+    this.onClick = this.onClick.bind(this);
+  }
+
+  state: State = {
+    active: false,
+  };
+
+  componentWillReceiveProps(nextProps: Props) {
+    if (nextProps.active !== this.props.active) {
+      this.setState({
+        active: nextProps.active,
+      });
+    }
+  }
+
+  private onClick() {
+    this.setState({
+      active: false,
```

Comment:
hmmm... this way, the parent component doesn't get notified when user clicks the banner to hide it.

I guess we don't need that for now (or the immediately foreseeable future), so that's fine for now :)